# Description: Don't include level4nl but only level4n for french oss layout.
# both space and Ctrl + space returned the same XLookupString, which prevents
# space using in some application using gtk_action_group_add_toggle_actions
# to setup Ctrl + space accelerator. We loose rctrl accelator but that's better
# than not having space in those applications and setup it manually will
# reintroduce https://bugs.launchpad.net/bugs/198759
# Ubuntu: https://launchpad.net/bugs/221112
# Upstream: http://bugs.freedesktop.org/show_bug.cgi?id=15804

Index: xkeyboard-config-1.8/symbols/fr
===================================================================
--- xkeyboard-config-1.8.orig/symbols/fr	2010-04-14 17:52:47.850615502 +0200
+++ xkeyboard-config-1.8/symbols/fr	2010-04-14 17:53:03.174604295 +0200
@@ -129,7 +129,7 @@
 
     include "latin"
     include "level3(ralt_switch)"
-    include "nbsp(level4nl)"
+    include "nbsp(level4n)"
     include "keypad(oss)"
 
     name[Group1]="France - Alternative";
